src/Commands/GeneratorCommand.php

```diff
@@ -76,10 +76,11 @@ public function getDefaultNamespace() : string
      * Get class namespace.
      *
      * @param \Nwidart\Modules\Module $module
+     * @param string $concrete
      *
      * @return string
      */
-    public function getClassNamespace($module)
+    public function getClassNamespace($module, string $concrete = null)
```

Comment:
Why is this argument needed? All other commands were able to make it work without it.

Reply:
Other commands work with one namespace, this command need to generate a few different namespaces.